pkg/network/listener.go

```diff
@@ -108,12 +107,6 @@ func (l *listener) Addr() net.Addr {
 }
 
 func (l *listener) Start(lctx context.Context, restart bool) {
-	defer func() {
-		if r := recover(); r != nil {
-			log.DefaultLogger.Alertf("listener.start", "[network] [listener start] panic %v\n%s", r, string(debug.Stack()))
```

Comment:
you can add this log into recover function

Reply:
sorry i don't way to add this log into revover function.
i think delete all of this is a good way, because it already output log with stack info in function `utils.GoWithRecover`.
In other places  of mosn, it doesn't print log repeatedly.

Reply:
utils.GoWithRecover print log to stderr, but we need this `Alertf`.